public/examples/integrations/toolkits/github/update_pull_request_example_call_tool.py

```diff
@@ -17,8 +17,8 @@
 client.auth.wait_for_completion(auth_response)
 
 tool_input = {
-    "owner": "ArcadeAI",
-    "repo": "hello-world",
+    "owner": "YourGitHubOrg",
+    "repo": "your-repo",
```

Comment:
update PR assumes you have a PR already that you can edit, so I used a non-working thing for this one.